cmd/celestia-appd/cmd/root.go

```diff
@@ -77,7 +78,13 @@ func NewRootCmd() *cobra.Command {
 				return err
 			}
 
+			// change tendermint config for celestia app
+			// set broadcast timeout to be 50 seconds in order to factor in long block times
+			// set max body bytes for transaction to be 8 Mb
+			// TODO: maybe we should make this timeout configurable?
 			tmCfg := tmcfg.DefaultConfig()
+			tmCfg.RPC.TimeoutBroadcastTxCommit = 50 * time.Second
```

Comment:
[Non Blocking][Question for reviewers]
Even in the latest networks, we're seeing inconsistent block times. Why not increase this value to more? Like 5 min or so?
Or, even pass it as a parameter to the command, with the default being long enough.